cryptoprotocol/src/main/java/org/cryptimeleon/incentivesystem/cryptoprotocol/model/IncentivePublicParameters.java

```diff
@@ -22,6 +22,14 @@
     @Represented
     BilinearGroup bg;
 
+    @NonFinal
+    @Represented
+    GroupElement g1Generator;
```

Comment:
Which groups do you mean? We do not need to store the first two groups of the BG in IncentivePublicParameters since they can be accessed via the bg field.

Reply:
I see, since we can only change the name of the generators, we should give it an expressive name. Good point, leave it as it is :+1: